CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and this library adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   `onCriticalErrorHandler` in case any error occurs 
 - Block synchronization logic now works above batch of blocks with size 1000 blocks instead of just 100 blocks 
   except the Zcash sandblasting period in which batch size of 100 blocks is still used
+- The internal `FileCompactBlockRepository.BLOCKS_METADATA_BUFFER_SIZE` constant has been raised from 10 to 1000 to 
+  match the block synchronization batch size. The overall speed-up of the entire block synchronization logic is about 
+  50%.
```

Comment:
The benchmarks did not test scanning across sandblasting, so we don't know if the 50% number holds up there. I would expect it to still be significant though.

Reply:
But I also suspect it will be less than 50% for sandblasting ranges, because I think the 50% we see here is actually partially due to #1548. It increased the batch size outside sandblasting ranges by 10x, which would both improve batch scanning speed and make the IO contention problem worse. My guess is that this PR, by fixing the IO contention problem, is unlocking more of the batch size increase improvement.